config/armada/config.yaml

```diff
@@ -6,6 +6,9 @@ corsAllowedOrigins:
   - http://localhost:8089
   - http://localhost:10000
 cancelJobsBatchSize: 1000
+pulsarSchedulerEnabled: false
+probabilityOfUsingPulsarScheduler: 0
+ignoreJobSubmitChecks: true
```

Comment:
I originally called it `submitJobSubmitChecks` but changed it to `ignore` because that's closer to what this actually does- the check is still performed it just ignores the result.  I wouldn't worry about this too much- it's only staying in for one release, once we have migrated everything over to the new check mechanism (and thus redis is populated) we can remove this flag.